src/components/Tabs/Annotation/AnnotationTab.tsx

```diff
@@ -145,6 +138,7 @@ export default function AnnotationTab(props: AnnotationTabProps): ReactElement {
     return {
       value: index.toString(),
       label: label.ids.size ? `${label.name} (${label.ids.size})` : label.name,
+      color: label.color,
```

Comment:
why isn't this optional? or sometimes undefined?

Reply:
It is an optional property, yes. It's always defined for labels (since the color is defined when the label is created) but in all other instances of `SelectionDropdown` it's currently undefined.